vision/object_detection_segmentation/ssd-mobilenetv1/README.md

```diff
@@ -0,0 +1,137 @@
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|SSD-MobilenetV1       | [29.3 MB](object_detection_segmentation/ssd-mobilenetv1/models/ssd_mobilenet_v1_coco_2018_01_28.onnx)  |[76.5 MB](object_detection_segmentation/ssd-mobilenetv1/models/ssd_mobilenet_v1_coco_2018_01_28.tar.gz) |1.7.0 | 10 |
+
+### Source
+Tensorflow SSD-MobileNetV1 ==> ONNX SSD-MobileNetV1 
+
+## Inference
+
+### Running inference 
+Refer to this [inference notebook](https://github.com/onnx/tensorflow-onnx/blob/master/tutorials/ConvertingSSDMobilenetToONNX.ipynb) for more details on how to run inference using onnxruntime and define environment variables for the model. 
+     
+    import onnxruntime as rt
+    
+    # load model and run inference
+    sess = rt.InferenceSession(os.path.join(WORK, MODEL + ".onnx"))
+    result = sess.run(outputs, {"image_tensor:0": img_data})
+    num_detections, detection_boxes, detection_scores, detection_classes = result
+    
+    # print number of detections
+    print(num_detections)
+    print(detection_classes)
+
+    # produce outputs in this order
+    outputs = ["num_detections:0", "detection_boxes:0", "detection_scores:0", "detection_classes:0"]
+
+
+### Input
+This model does not require fixed image dimension. Input batch size is 1, with 3 color channels. Image has these variables: `(batch_size, height, width, channels)`. 
+
+### Preprocessing
+The following code shows how preprocessing is done. For more information and an example on how preprocess is done, please visit the [inference notebook](https://github.com/onnx/tensorflow-onnx/blob/master/tutorials/ConvertingSSDMobilenetToONNX.ipynb).
+
+    import numpy as np
+    from PIL import Image, ImageDraw, ImageColor
+    import math
+    import matplotlib.pyplot as plt
+    
+    # open and display image file
+    img = Image.open("image file")
+    plt.axis('off')
+    plt.imshow(img)
+    plt.show()
+    
+    # reshape the flat array returned by img.getdata() to HWC and than add an additial    
+    dimension to make NHWC, aka a batch of images with 1 image in it
+    img_data = np.array(img.getdata()).reshape(img.size[1], img.size[0], 3)
+    img_data = np.expand_dims(img_data.astype(np.uint8), axis=0)
+ 
+### Output
+Given each batch of images, the model returns 4 tensor arrays:
+
+`num_detections`: the number of detections.
+
+`detection_boxes`: a list of bounding boxes. Each list item describes a box with top, left, bottom, right relative to the image size.
+
+`detection_scores`: the score for each detection with values between 0 and 1 representing probability that a class was detected.
+
+`detection_classes`: Array of 10 integers (output as floating point values) each indicating the index of a class label from the coco class
+
+It outputs the image with boundary boxes and labels. The full list of classes can be found in the [coco dataset](https://cocodataset.org/#home).
```

Comment:
```suggestion
It outputs the image with boundary boxes and labels. The full list of classes can be found in the [COCO dataset](https://cocodataset.org/#home).
```

correct coco to COCO in other areas in this readme as well
move this sentence to the beginning of the section